std/typecons.d

```diff
@@ -444,13 +455,29 @@ template Tuple(Specs...)
         }
 
         /**
-         * Use $(D t.expand) for a tuple $(D t) to expand it into its
-         * components. The result of $(D expand) acts as if the tuple components
+         * Use $(D t.expand) for a `Tuple` $(D t) to expand it into its
+         * components. The result of $(D expand) acts as if the `Tuple's` components
```

Comment:
I agree, but I also think it's important that anything inside of the \`\` be actual types or code.